config/plug/utils/markview.nix

```diff
@@ -2,12 +2,12 @@
   extraPlugins = with pkgs.vimUtils; [
     (buildVimPlugin {
```

Comment:
Oops I forgot to add `rec` @elythh can you fix that?

Reply:
pushed 😄

Reply:
I should add a CI job